package.json

```diff
@@ -29,22 +29,22 @@
     "moment": "2.21.0",
     "q": "1.5.1",
     "regression": "2.0.1",
-    "underscore": "1.8.3"
+    "underscore": "^1.13.1"
   },
   "devDependencies": {
-    "@silvermine/eslint-config": "3.0.0-rc.0",
-    "@silvermine/standardization": "1.0.1",
+    "@silvermine/eslint-config": "3.0.1",
+    "@silvermine/standardization": "1.1.0",
     "aws-sdk": "2.354.0",
     "check-node-version": "4.0.3",
     "coveralls": "3.0.2",
     "eslint": "6.8.0",
     "expect.js": "0.3.1",
-    "grunt": "1.0.3",
+    "grunt": "^1.4.0",
```

Comment:
Looks like a `^` slipped in.

Reply:
@jimjenkins5 this doesn't seem to have been fixed?